hypha/apply/home/migrations/0002_add_apply_homepage.py

```diff
@@ -29,7 +29,7 @@ def create_homepage(apps, schema_editor):
 
     # Create a site with the new homepage set as the root
     Site.objects.create(
-        hostname="apply.localhost", root_page=applyhomepage, is_default_site=False
+        id=2, hostname="apply.localhost", root_page=applyhomepage, is_default_site=False
```

Comment:
This migration will only run for fresh setup and not those where the migrations has already been run. 

Is the intension here to fix these IDs for the existing setups? If so, it would need a fresh migration with custom python code.